cabal-install/Distribution/Client/GenBounds.hs

```diff
@@ -131,7 +133,8 @@ genBounds verbosity packageDBs repoCtxt comp platform progdb mSandboxPkgInfo
                   verbosity packageDBs repoCtxt comp platform progdb
                   mSandboxPkgInfo globalFlags freezeFlags
 
-       putStrLn boundsNeededMsg
+       hPutStrLn stderr boundsNeededMsg
+       hFlush stderr
```

Comment:
I added it as a precaution because I saw `stdout` being used a few lines later.  I've removed it.

I *think* I saw stderr being set to line-buffered in the code somewhere, but I don't know for sure.  But if that is the case then the flush shouldn't be needed.